models/eos/interfaces/merged_example_01.txt

```diff
@@ -0,0 +1,44 @@
+
+# Using merged
+
+# Before state:
+# -------------
+#
+# veos#show running-config | section interface
+# interface Ethernet1
+#    description "Interface 1"
+# !
+# interface Ethernet2
+# !
+# interface Loopback1
+# !
+# interface Management1
+#    ip address dhcp
+# !
+
+- name: Configure interfaces
```

Comment:
Same comment as above, maybe something like "Merge the provided configuration for Ethernet1 and Ethernet2 with the current configuration on the network device"